### PR TITLE
Danger blog post

### PR DESCRIPTION
**Pull request by annabatki**
Should go live on Friday.
Needed: images

### CODE REVIEW

**Comment by annabatki** [comment]
This is ready to be reviewed!

File: source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+([Source](http://danger.systems/ruby/))
+
+##Setting it all up!
+
+> [__Bundler__](http://bundler.io/) is a dependency manager for Ruby which uses a Gemfile to define all of the Ruby projects you want to use.
+
+Let's open up your Terminal app and head to your project's path. If you don't have Bundler installed to your system, type in the following command.
+
+<pre><code># If you need to install bundler
+gem install bundler
```
Comment (viktorbenei) [inline source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown]: might require `sudo` - if MacOS default/system Ruby is used

File: source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+Also while we are here, let's add the `Danger` gem to the Gemfile. We will use Ruby for `Danger`, however you can use [Javascript](http://danger.systems/js/) as well for the setup.
+
+<pre><code># Create a Gemfile
+bundler init
+
+# Adding the Danger gem to the Gemfile
+echo "gem 'danger'" >> Gemfile
+</code></pre>
+
+You can also open the Gemfile in your favorite editor and remove `#gem 'rails'` to keep it nice and tidy.
```
Comment (viktorbenei) [inline source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown]: add an example Gemfile here

File: source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown (hunk 1 above)
Comment (annabatki) [inline source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown]: @birmacher ?

File: source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown (hunk 2 above)
Comment (annabatki) [inline source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown]: @birmacher ?

File: source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown (hunk 1 above)
Comment (birmacher) [inline source/2017-07-27-danger-danger-uh-that-is-using-danger-with-bitrise.html.markdown]: I'd leave it as is

**Pull request merged**